elements/cni-plugins/pkg-map

```diff
@@ -0,0 +1,8 @@
+{
+  "family": {
+    "redhat": {
+      "arping": "iputils",
+      "iptables": "iptables iptables-libs iptables-legacy iptables-legacy-libs"
```

Comment:
@mnaser If we want to have iptables and -legacy stuff here - we need EPEL in place before that.
One thing that worries me that pkg-mapping is very loose when it comes to dependencies. I found that if packages is not found - it's just skipped.

Reply:
Oh that's annoying. I think epel goes into pre install element. Let me check.